examples/factors/README.md

```diff
@@ -2,11 +2,24 @@
 
 The _factors_ example is a minimalistic RISC Zero zkVM proof. The prover demonstrates that they know two nontrivial factors (i.e. both greater than 1) of a number, without revealing what those factors are. Thus, the prover demonstrates that a number is composite — and that they know the factors — without revealing any further information about the number.
 
-To see this example in action, [install Rust] and use `cargo run` in this directory to run it:
+First, [install Rust] if you don't already have it. 
+
+Next, install the `cargo-risczero` tool and install the toolchain with:
+
+```bash
+cargo install risczero
```

Comment:
Oops, this should be:
```
cargo install cargo-risczero
```

I realized I probably made this typo in all the other examples

Reply:
hmm... i updated the command in bevy, chess, and factors.
I thought I saw it in more changes when I reviewed your PR from earlier, but this is all I'm seeing right now